v2/tools/generator/internal/config/object_model_configuration.go

```diff
@@ -244,6 +244,33 @@ func (omc *ObjectModelConfiguration) VerifyAzureGeneratedConfigsConsumed() error
 	return visitor.Visit(omc)
 }
 
+// ManualAzureGeneratedConfigs looks up a type to determine if it has any Azure generated configs
+func (omc *ObjectModelConfiguration) ManualAzureGeneratedConfigs(name astmodel.TypeName) ([]string, error) {
```

Comment:
I'm confused by this name. How can something be _manual_ and _generated_ at the same time?

Reply:
So basically what I am trying to evoke here is `$azureGeneratedSecrets` equivalent, which is that the code-generator will generate the property on `spec.OperatorSpec.ConfigMaps.<YourPropertyNameHere>`, but nothing will ever populate it unless you write a manual extension.

This is exactly how `$azureGeneratedSecrets` works too.

Maybe what we should do is call this `$manualConfigs` and drop the `AzureGenerated`?

Reply:
Did the above. Also changed `$azureGeneratedConfigs` to just `$generatedConfigs`